test/util/test_llarp_util_decaying_hashset.cpp

```diff
@@ -2,11 +2,42 @@
 #include <router_id.hpp>
 #include <catch2/catch.hpp>
 
+TEST_CASE("Thrash DecayingHashSet", "[decaying-hashset]")
+{
+  static constexpr auto duration = 5s;
+
+  static constexpr auto decayInterval = 50ms;
+  llarp::util::DecayingHashSet<llarp::AlignedBuffer<32>> hashset(decayInterval);
+  const llarp_time_t started = llarp::time_now_ms();
+  const auto end = duration + started;
+  llarp_time_t nextDecay = started + decayInterval;
+  do
+  {
+    const auto now = llarp::time_now_ms();
+    for (size_t i = 0; i < 500; i++)
+    {
+      llarp::AlignedBuffer<32> rando;
+      rando.Randomize();
+      hashset.Insert(rando, now);
+      /// maybe reinsert to simulate filter hits
+      if (i % 20 == 0)
+        hashset.Insert(rando, now);
+    }
+    if (now >= nextDecay)
+    {
+      REQUIRE(not hashset.Empty());
+      hashset.Decay(now);
+      nextDecay += decayInterval;
+    }
+
+  } while (llarp::time_now_ms() <= end);
```

Comment:
This seems like more of a stress test... a few of these aren't a big deal, but when you have a lot of tests that run for specific amounts of time, it can add up. Maybe we need a set of tests that only run on CI builds and/or as a special make target.